src/app/view/components/_common/dot-push-publish-env-selector/dot-push-publish-env-selector.component.html

```diff
@@ -13,7 +13,7 @@
 >
 </p-multiSelect>
 
-<ul class="environment-selector__list">
+<ul class="environment-selector__list" *ngIf="showList">
```

Comment:
Why is this need it now?

Reply:
didn't want to remove the functionality of show the list, instead added a flag to make it visible if needed. by default will be false.